packages/babel-types/scripts/generators/docs.js

```diff
@@ -104,8 +112,47 @@ Object.keys(t.BUILDER_KEYS)
         } else {
           fieldDescription.push(" (required)");
         }
-        readme.push(" - " + fieldDescription.join(""));
+        readme.push("- " + fieldDescription.join(""));
       });
+  }
+}
+
+function printAliasKeys(key, readme) {
+  if (t.ALIAS_KEYS[key] && t.ALIAS_KEYS[key].length) {
+    readme.push("");
+    readme.push(
+      "Aliases: " +
+        t.ALIAS_KEYS[key]
+          .map(function (key) {
+            return "`" + key + "`";
+          })
+          .join(", ")
+    );
+  }
+}
+
+Object.keys(t.BUILDER_KEYS)
+  .sort()
+  .forEach(function (key) {
+    readme.push("### " + toFunctionName(key));
```

Comment:
Looking at the rendered part again,

maybe in another PR, we should have a quick section above that explains the terms "builder" and the difference between

t.anyTypeAnnotation();
t.isAnyTypeAnnotation(node, opts)
t.assertAnyTypeAnnotation(node, opts).

I would just mention that instead of manually creating the AST node as an object, we provide this interface with some assertions etc. And then the other two methods as an alias t.isX = node.type === "x", and same with assert?

Reply:
Not sure where we would do it, but it would also be nice to have a short snippet of what that code even looks like since this page assume you know what each node is. We can also link to astexplorer as a way to test it.

Reply:
> have a quick section above that explains the terms "builder" and the difference between
> 
> t.anyTypeAnnotation();
> t.isAnyTypeAnnotation(node, opts)
> t.assertAnyTypeAnnotation(node, opts).

Agreed. I have split the "API" section, tentatively, to "Node Builders" and "Aliases" in https://github.com/babel/babel/pull/13151, maybe we can add new sections about the usage of `is*` and `assert*`.